test/core/libp2p/host_builder_test.cpp

```diff
@@ -10,19 +10,13 @@
 using namespace libp2p;
 using namespace libp2p::multi;
 
-class HostBuilderTest : public ::testing::Test {
- public:
-  const Multiaddress kDefaultAddress =
-      Multiaddress::create("/ip4/192.168.0.1").value();
-
-  boost::asio::io_context io_context_;
-};
+class HostBuilderTest : public ::testing::Test {};
 
 /**
  * @given host builder
  * @when building a host with no parameters
  * @then build does not fail
  */
 TEST_F(HostBuilderTest, AllDefaults) {
-  HostBuilder{io_context_}.build();
+  EXPECT_TRUE(HostBuilder{}.build());
```

Comment:
nice test

Reply:
I think, it's going to be extended, when some of the Host's parts are implemented, and we will have more ways to check validity of the built object